module/proto/gravitynft/v1/attestation.proto

```diff
@@ -16,11 +16,9 @@ enum NFTClaimType {
   // A claim for a SendNFTToCosmos transaction
   NFT_CLAIM_TYPE_SEND_NFT_TO_COSMOS      = 1;
 
-  // TODO: WILL WE USE THIS?
   // A claim for when an NFT is sent to Ethereum
   NFT_CLAIM_TYPE_SEND_NFT_TO_ETH   = 2;
 
-  // TODO: ASK JAKUB ABOUT THIS ONE: IS THIS THE NFT-GRAVITY CONTRACT?
```

Comment:
This is for ERC721 contract deployed by gravity when we're sending cosmos-originating nft to Ethereum